libs/novui/src/types/CoreProps.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Defines the foundational props from which all Novu components should inherit.
+ */
+export interface CoreProps {
+  className?: string;
+  'data-test-id'?: string;
```

Comment:
I'd like to suggest removing `data-test-id` and promote accessible selectors. Mantine v7 is stronger in that compared to v5. The team should strive to eliminate the need for data-test-id.

Reply:
Sounds good to me -- I have not been a fan of our use of it either